packages/storage-adapters/resolve-storage-mongo/package.json

```diff
@@ -24,7 +24,9 @@
   "devDependencies": {
     "chai": "^3.5.0",
     "jest": "^22.1.4",
-    "sinon": "^3.3.0"
+    "sinon": "^3.3.0",
+    "bluebird": "^2.9.34",
```

Comment:
Please, clarify, why you add the 'bluebird' package? Is it a syntax sugar or really necessary thing?